Tribler/Test/util/Tracker/UDPTracker.py

```diff
@@ -42,11 +45,11 @@ def datagramReceived(self, response, host_and_port):
                 self.send_error(host, port, "no infohash")
                 return
 
-            num_infohashes = (len(response) - 16) / LENGTH_INFOHASH
+            num_infohashes = (len(response) - 16) // LENGTH_INFOHASH
```

Comment:
Import `division`?

Reply:
⏫ This is required.

Reply:
Why? The `//` operator refers to integer division in both Python 2 and 3.